lldb/test/API/commands/dwim-print/swift/main.swift

```diff
@@ -6,9 +6,17 @@ class Object: CustomStringConvertible {
   }
 }
 
+class User {
+    var id: Int = 314159265358979322
+    var name: String = "Gwendolyn"
+    var groups: (admin: Bool, staff: Bool) = (false, true)
+}
+
 func main() {
     let object = Object()
-    _ = object // break here
+    let user = User()
+    // break here
```

Comment:
`print("break here")` is safer.

Reply:
I was under the impression that the following line (`_ = (object, user)`) has the same safety effect as the `print` would. Is that not the case?

Reply:
I'm sure it works fine now. A (future) compiler could decide to optimize away the assignment, but not an i/o call.